setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup
 
 setup(name='vm_automation',
-      version='0.1.2',
+      version='0.1.3',
```

Comment:
No bump here `0.1.2` is the currently in development version.

Reply:
OK, wasn't sure since I was adding a dependency and keyword.  Will un-bump.